src/Orckestra.Composer.Cart/Services/Order/IOrderService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Orckestra.Composer.Cart.ViewModels.Order;
+using System;
+using System.Threading.Tasks;
+
+namespace Orckestra.Composer.Cart.Services.Order
+{
+    public interface IOrderService
```

Comment:
I suggested it would be great to have a separate general order service since the `OrderHistoryViewService` is something related to the order history, but our changes aren't. Are we sure about the `ViewService` naming? For example, `CartService` is named without this `ViewService` part, but returns view models in many places. Also, in the future, we can expect, that this service will return not only the view models but something else. If we expect to have separate `OrderViewService` and `OrderService` in that case, it sounds good to rename. What do you think?

Reply:
It will be used in both Order Details and Order History pages, for that reason we have one ViewService
And yes, we have practice to name services as ViewServices as they return ViewModel.
CartService is one of not correct example - at some point nedd also rename it